generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>Can we break down the section markdown:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">## Environment settings

This step include the path setup of data directory, model directory, output directory, and etc.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Would you help explain more on it? Thanks